src/implementations/twig/components/social-media-follow/social-media-follow.deprecated.story.js

```diff
@@ -0,0 +1,119 @@
+import { withNotes } from '@ecl/storybook-addon-notes';
+import { correctPaths } from '@ecl/story-utils';
+import withCode from '@ecl/storybook-addon-code';
+
+import specsColor from '@ecl/specs-component-social-media-follow/demo/data--color';
+import SocialMediaFollow from './social-media-follow.html.twig';
+import notes from './README.md';
+
+// Preserve original data.
+const dataHorizontalColor = { ...specsColor };
+const dataVerticalColor = { ...specsColor, variant: 'vertical' };
+
+const getArgs = (data) => ({
+  show_other: true,
+  show_label: true,
+  position: 'left',
+  description: data.description,
+});
+
+const getArgTypes = () => ({
+  show_label: {
+    name: 'label',
+    type: { name: 'boolean' },
+    description: 'toggle the visibility of the links label',
+    table: {
+      category: 'Optional',
+    },
+  },
+  show_other: {
+    name: 'other social networks',
+    type: { name: 'boolean' },
+    description: 'toggle the visibility of the "other social networks" link',
+    table: {
+      category: 'Optional',
+    },
+  },
+  position: {
+    name: 'position',
+    type: 'select',
+    description: 'Position',
+    options: ['left', 'right'],
+    control: {
+      labels: {
+        left: 'left',
+        right: 'right',
+      },
+    },
+    mapping: {
+      left: 'left',
+      right: 'right',
+    },
+    table: {
+      type: 'string',
+      defaultValue: { summary: 'left' },
+      category: 'Display',
+    },
+  },
+  description: {
+    name: 'description',
+    type: { name: 'string', required: true },
+    description: 'The description of the elements',
+    table: {
+      type: { summary: 'string' },
+      defaultValue: { summary: '' },
+      category: 'Content',
+    },
+  },
+});
+
+const prepareData = (data, args) => {
+  correctPaths(data);
+  const clone = JSON.parse(JSON.stringify(data));
+
+  if (!args.show_other) {
+    delete clone.links.pop();
+  }
+
+  if (!args.show_label) {
+    clone.links.forEach((element) => {
+      element.link.hide_label = true;
+    });
+  }
+
+  return Object.assign(clone, args);
+};
+
+export default {
+  title: 'Components/Social Media Follow',
```

Comment:
we are missing the 'deprecated' here, so the stories are still visible in the component

Reply:
added, thanks.. i missed that.